tests/functional/test_data_completeness.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from botocore.session import Session
+from botocore.loaders import Loader
+from botocore.exceptions import DataNotFoundError
+
+
+class TestDataTypes(object):
+
+    def _test_data_type_is_not_lost(self, service_name, type_name,
```

Comment:
Can we switch ``data_type`` naming convention to ``model``. When I think of ``data_type`` I think of the actual data type passed to a parameter and the not the model type.